documentation/develop/getting-started-with-web-ext.md

```diff
@@ -185,11 +185,11 @@ You can connect to this Android device on TCP port 51499
 
 This is a remote debugger port that you can [connect to with Firefox's developer tools](https://developer.mozilla.org/en-US/docs/Tools/Remote_Debugging/Firefox_for_Android#Connecting). In this case, you'd connect to host `localhost` on port `51499`.
```

Comment:
Should not we remove the locale from the rest of the urls in this file or need to file a bug to remove all locale at once from the project @caitmuenster

Reply:
👍 Yes, good catch -- addressing it in next commit